### DISCUSSION TITLE
Number of workers

### DISCUSSION THREAD

**Discussion by K2ouMais** [serversideup/docker-php#90] [Q&A] [answered]
Hello everyone,

I am thinking in using your images for my projects.

All my projects are using queues, but I couldn't find anything about on to set the amount of workers that are picking up jobs.

Seems that your container only uses a single worker when running PHP queue:work.

How do you ensure that the command doesn't crash?

Thank you I advance

**Comment by jaydrogers** [comment, marked as answer]
Thanks for stopping by @K2ouMais!

What I would do in that case is keep 1 worker process per container, then use the orchestrator (like Swarm or K8s) to scale the job.

If the process fails and exits with a non-zero exit code, the container will die.

Hope this helps 👍
